code/core/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt

```diff
@@ -218,6 +215,26 @@ internal class EventHub(val eventHistory: EventHistory?) {
             ).get()
         }
 
+    /**
+     * Initializes event history. This must be called after the SDK has application context.
+     */
+    fun initializeEventHistory() {
+        if (eventHistory != null) {
+            return
```

Comment:
nit: Can we add a log here to allow identifying any instance attempting a set twice?